ecm-wcm-extension/src/main/webapp/javascript/eXo/ecm/document-preview.js

```diff
@@ -203,6 +211,36 @@
         likeLink.tooltip('show');
       }
     },
+    fetchLiked: function() {
+      var self = this;
+      return $.ajax({
+        type:'GET',
+        url: '/rest/v1/social/activities/' + this.settings.activity.id + '/likes/' + eXo.env.portal.userName
+      }).done(function (data) {
+        if (data.likes != null) {
+          self.settings.activity.liked = true;
+        }
+        self.clearErrorMessage();
+      }).fail(function () {
+        self.settings.activity.liked = false;
+      });
+    },
+
+    fetchActivity: function() {
+      var self = this;
+      return $.ajax({
+        type:'GET',
+        url: '/rest/v1/social/activities/' + this.settings.activity.id
```

Comment:
To make it go through /portal and not create a session for /rest
```suggestion
        url: '/portal/rest/v1/social/activities/' + this.settings.activity.id
```